### DISCUSSION TITLE
GITHUB_ACTIONS env variable is not set in container

### DISCUSSION THREAD

**Discussion by dawidd6** [community/community#25592] [Actions]
<p>See this example workflow:</p>
<p><a href="https://github.com/dawidd6/homebrew-tap/runs/236440509" rel="nofollow noopener">https://github.com/dawidd6/homebrew-tap/runs/236440509</a></p>

**Comment by ethomson** [comment, marked as answer]
<p>Thanks for reporting this, we’ll take a look.</p>

**Comment by dawidd6** [comment]
<p>Seems like it’s resolved now, thanks!</p>

**Comment by atifsyedali** [comment]
<p>Still doesn’t work for $HOME.</p>
